pkgs/development/python-modules/pytest-watch/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, pkgs
+, pythonPackages
+, buildPythonPackage
+, fetchPypi
+, docopt
+, colorama
+, pytest
+, watchdog
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-watch";
+  version = "4.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06136f03d5b361718b8d0d234042f7b2f203910d8568f63df2f866b547b3d4b9";
+  };
+
+  # No Tests
+  doCheck = false;
+  buildInputs = [ pytest ];
```

```suggestion
  checkInputs = [ pytest ];
```

`nativeBuildInputs` that only get added when `doCheck = true;`

---

Does not build without `pytest`. I could use `nativeBuildInputs`, but it would be pointless to have `pytest-watch` without `pytest`, right?